slack-api-client/src/test/java/test_with_remote_apis/methods/files_Test.java

```diff
@@ -1167,4 +1167,91 @@ public void filesUploadV2_manual_two_files() throws Exception {
         FilesInfoResponse file1info = client.filesInfo(r -> r.file(file1Upload.getFileId()));
         assertThat(file1info.getFile().getShares().getPublicChannels().get(randomChannelId), is(notNullValue()));
     }
+
+    @Test
+    public void issue1345_filesUploadV2_multiple() throws IOException, SlackApiException {
+        loadRandomChannelId();
+        MethodsClient slackMethods = slack.methods(userToken);
+
+        byte[] fileData = Files.readAllBytes(Paths.get("src/test/resources/sample.txt"));
+
+        // note: this test can be done using only one file as everything tested applies to each file individually - the only important thing is to use "uploadFiles" instead of direct values
+        FilesUploadV2Request.UploadFile uploadFile = FilesUploadV2Request.UploadFile.builder().filename("issue1345_test.txt").title("issue1345 test").fileData(fileData);
```

Comment:
```suggestion
        FilesUploadV2Request.UploadFile uploadFile = FilesUploadV2Request.UploadFile.builder().filename("issue1345_test.txt").title("issue1345 test").fileData(fileData).build();
```

This seems to be causing [compilation errors](https://github.com/slackapi/java-slack-sdk/actions/runs/10402312991/job/28880285507?pr=1346#step:4:434) right now 🤔

Reply:
whoops, forgot calling `#build` on each of these builders

Reply:
should be fixed now